### PR TITLE
(MAINT) Prep for 1.2.0 release

### PR DESCRIPTION
**Pull request by cprice404**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
+## 1.2.0
+
+This is a feature release.
+
+* Upgrade to version 9.2.8 of upstream Jetty (we were previously at
+  v9.1.0, which was over a year old).
```
Comment (KevinCorcoran) [inline CHANGELOG.md]: Is there any more-specific reason for the upgrade?

**Comment by KevinCorcoran** [comment]
:+1: but if there is anything else that can be said about the reason for upgrading the Jetty dependency, it should be in the CHANGELOG.

**Comment by cprice404** [comment]
@KevinCorcoran  Added a little more verbiage.

**Comment by dankreek** [comment]
:+1:

**Pull request merged**